tasks/lib/run.js

```diff
@@ -104,9 +105,79 @@ RunUtil.displayScriptOutput = function(grunt, outputs, data) {
   }
 };
 
+function getConstructorName(value) {
+  if(value == null) { // is it undefined or null
+    return String(value);
+  }
+
+  return value.constructor.name;
+}
+
+function getFallbackErrorMessage(err) {
+  switch(err.__fxotype__) {
+    case 'AuthError':
+      return 'The request failed because of an authorization problem.';
+    case 'ServiceError':
+      return 'There was an error with your task, please contact support.';
+    case 'WebhookIgnoreError':
+      return 'The data received by the webhook was ignored.';
```

Comment:
Yes, currently the core will retry. I've added a Trello card to fix that,
which will need to be done prior to any service going live which will use
this feature.